src/config.ts

```diff
@@ -8,7 +8,7 @@ import * as util from '@cmt/util';
 import * as os from 'os';
 import * as vscode from 'vscode';
 
-export type LogLevelKey = 'trace'|'debug'|'info'|'note'|'warning'|'error'|'fatal';
+export type LogLevelKey = 'trace' | 'debug' | 'info' | 'note' | 'warning' | 'error' | 'fatal';
```

Comment:
We would like for formatting changes to be done as a separate PR.  I was planning to do a full re-formatting pass on the extension once we clear out the outstanding PR's so that we don't have to deal with lots of merge conflicts.  It also makes it really hard to see what "real" changes you've actually made.

Reply:
Understandable, will try to get those back in original state again! I had format on save active so it happened automatically :-)

Reply:
Formatting should be ok now.